### ISSUE TITLE
This work is crazy, hope the code gets released. 

### ISSUE THREAD

**Issue by JaeDukSeo** [xamyzhao/timecraft#1]
This is awesome work, hope to play around with it.

**Comment by xamyzhao** [comment]
Thank you! I'm working on cleaning up the code for release and will hopefully have it up within a few weeks.

**Comment by Vaelek** [comment]
How's that looking?

**Comment by xamyzhao** [comment]
@Vaelek it is coming along! I have the model pre-training and evaluation code working, and I am cleaning up and debugging the sequential training code. If you are interested in trying out the model, I can commit that first.

**Comment by ghost** [comment]
@xamyzhao I would also be interested in trying out the model, great work thanks

**Comment by xamyzhao** [comment]
Just uploaded the trained models and some sample inference code! Please let me know if you run into any issues with it. I'll be working on cleaning up the rest of the training code.

**Comment by ghost** [comment]
@xamyzhao for inference,  cv2.imshow requires an X server, which isn't available in google colab

**Comment by ghost** [comment]
should this line 84-85 be like this in make_timelapse.py
        pred_vid_im = vis_utils.visualize_video(pred_vid[0], normalized=True) * 255.0

**Comment by xamyzhao** [comment]
@ak9250 feel free to save or display the images however you wish; the provided script is just some sample code that shows how to run the model. `cv2.imshow` displays images in the range [0, 1] just fine for me, but you can convert the images to [0,255] uint8 if you wish.
